api/include/mfxstructures.h

```diff
@@ -1689,11 +1716,41 @@ typedef struct {
     union {
         mfxU8  *QP;         // Block QP value. Valid when Mode = MFX_MBQP_MODE_QP_VALUE
         mfxI8  *DeltaQP;    // For block i: QP[i] = BrcQP[i] + DeltaQP[i]. Valid when Mode = MFX_MBQP_MODE_QP_DELTA
+#if (MFX_VERSION >= 1034)
+        mfxQPandMode *QPmode; // Block-granularity modes when MFX_MBQP_MODE_QP_ADAPTIVE is set
```

Comment:
I know that this is union, but still we have both ```QP``` and ```DeltaQP``` in static asserts file, let's also add ```QPmode```

Reply:
ok